tools/scylla-nodetool.cc

```diff
@@ -1776,11 +1777,44 @@ void snapshot_operation(scylla_rest_client& client, const bpo::variables_map& vm
     fmt::print(std::cout, "Snapshot directory: {}\n", params["tag"]);
 }
 
+static std::vector<sstring> rjson_array_to_sstring_vector(const rjson::value& v) {
+    std::vector<sstring> result;
+    result.reserve(v.Capacity());
+
+    for (auto& element : v.GetArray()) {
+        result.emplace_back(element.GetString());
+    }
+    return result;
+}
+
+static bool keyspace_uses_tablets(scylla_rest_client& client, const sstring& keyspace) {
+    const std::unordered_map<sstring, sstring> params = {{"replication", "tablets"}};
+    const auto tablet_keyspaces = rjson_array_to_sstring_vector(client.get("/storage_service/keyspaces", params));
+    const auto found_it = std::find(std::begin(tablet_keyspaces), std::end(tablet_keyspaces), keyspace);
+    return found_it != std::end(tablet_keyspaces);
```

Comment:
This can be done much simpler:
```c++
auto res = client.get("/storage_service/keyspaces", params);
auto ks_array = res.GetArray();
return std::find_if(ks_array.begin(), ks_array.end(), [&] (const auto& json_ks) { return rjson::to_string_view(json_ks) == keyspace; }) != ks_array.end();
```

Reply:
Fixed. I also removed `rjson_array_to_sstring_vector()` - it is not needed now.